plugins-fixed/invada.lv2/inv_delay.ttl

```diff
@@ -78,6 +78,7 @@ Modeled by Invada
 	lv2:minimum 2.0 ;
 	lv2:maximum 200.0 ;
 	units:unit units:s ;
+    lv2:portProperty  mod:tempoRelatedDynamicScalePoints;
```

Comment:
please pay attention to spaces vs tabs.
note that you can also have multiple port properties in a single line, like this:
```
lv2:portProperty pprops:logarithmic , mod:tempoRelatedDynamicScalePoints ;
```